spec/controllers/catalog_controller_spec.rb

```diff
@@ -6,70 +6,95 @@
 #   log_in_as_mock_user(subject)
     @druid = 'rn653dy9317'  # a fixture Dor::Item record
     @item = instantiate_fixture(@druid, Dor::Item)
-    @user = double(User, :login => 'sunetid', :logged_in? => true, :is_admin => false, :is_viewer => false)
+    @user = User.find_or_create_by_webauth double('WebAuth', :login => 'sunetid', :logged_in? => true, :attributes => {'DISPLAYNAME' => 'Example User'}, :privgroup => "")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.